docs/docs/sourcing-from-forestry.md

```diff
@@ -0,0 +1,168 @@
+
+```json:title=package.json
+"scripts": {
+  "forestry:preview": "gatsby develop -p 8080 -H 0.0.0.0"
+},
+```
+
+For preview to work on Forestry's dashboard, port 8080 must be used and all network interfaces have to be bound to `0.0.0.0`. For more information on Forestry's previews go to the [Forestry.io docs](https://forestry.io/docs/previews/instant-previews/#adding-an-instant-preview).
+
+_Note_: The `forestry:preview` script can be named anything, but it must match the `instant_preview_command` setting in `.forestry/settings.yml`. This sets up Forestry to run a local server so that CMS content can be previewed from the Forestry dashboard using `gatsby develop`.
+
+### CMS Content Configuration
+
+Now you can set up a content collection. Create the file `.forestry/front_matter/blog.yml` and paste in the following:
+
+```yml:title=.forestry/front_matter/blog.yml
+---
+label: Blog
+hide_body: false
+fields:
+  - name: date
+    type: datetime
+    label: Date
+    description: ""
+    config:
+      required: true
+      date_format:
+      time_format:
+      display_utc: false
+    default: now
+  - name: title
+    type: text
+    label: Title
+    description: ""
+    config:
+      required: true
+```
+
+Next, link the frontmatter collection in `.forestry/settings.yml`. This adds a template in the Forestry dashboard that allows you to add new Blog posts with the configured fields in the CMS.
+
+```yml:title=.forestry/settings.yml
+---
+new_page_extension: md
+auto_deploy: false
+admin_path: /static/admin
+webhook_url:
+sections:
+upload_dir: static/uploads
+public_path: "/static/uploads"
+front_matter_path: ''
+use_front_matter_path: false
+file_template: ":filename:"
+build:
+  preview_output_directory: public
+  install_dependencies_command: npm install
+  preview_docker_image: node:10
+  mount_path: "/srv"
+  working_dir: "/srv"
+  instant_preview_command: npm run forestry:preview
+sections:
+- type: directory
+  path: src/content/posts
+  label: Posts
+  create: documents
+  match: "**/*"
+  templates:
+  - blog
+```
+
+Because Forestry's `admin.html` file is in the `/static` directory, this page will only be available once your Gatsby site is built. Run `gatsby build && gatsby serve`. Once the server is running, it will print the address to open for viewing. It's typically localhost:8000. Now navigate to /admin - so if your site is at localhost:8000, go to localhost:8000/admin.
+
+You should now be viewing your Forestry login page. Login to view your dashboard. If you don't have a user yet, create one on the Forestry Dashboard. This will allow you to login on the Gatsby site admin login page. On the left side of your dashboard you will see "Posts". If you click on it you can add a new post. When you save your post, Forestry will push the change directly to your GitHub repo.
+
+### Making Changes
+
+Alright - you're all set to make changes in Netlify CMS and see them as commits in your GitHub repo!
```

Comment:
```suggestion
Alright - you're all set to make changes in Netlify CMS and see them as commits in your GitHub repo!
```
It's no longer Netlify CMS. Should this reference adding new files to your Gatsby repo?